staff/nacho/task-client-server/src/index.js

```diff
@@ -0,0 +1,56 @@
+require('dotenv').config()
+
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const axios = require('axios')
+
+app.set('view engine', 'pug')
+
+//Debemos efectuar una llamada a la API, para lo cual utilizaremos axios.
+//app.get para que por defecto muestre el index, luego llamamos a la api con axios a show/tasks, que es la url de nuestra api que devuelve el listado de tareas
+//finalmente lo renderizamos
+app.get('/', (req, res) => {
+    axios.get('http://localhost:5000/api/show/tasks').then(object =>{
+        res.render('index', { todos: object.data, dones: object.data })
+    })
+
+    // res.render('index', {tasks})
+})
+
+const formBodyParser = bodyParser.urlencoded({ extended: false })
+
+app.post('/tasks', formBodyParser, (req, res) => {
+    const { body: { text } } = req
+    const id = new Date().getTime()
+    const done = false
+    const username = "x"
```

Comment:
you only need to pass req.body.text, the other components will be create in the server side